odl/util/utility.py

```diff
@@ -417,6 +417,33 @@ def normalized_index_expression(indices, shape, int_to_slice=False):
     return tuple(indices)
 
 
+def snr(signal, noise, impl):
+    """Compute the signal-to-noise ratio.
+    This compute::
```

Comment:
This doc needs to be updated to conform to the ODL standard.

http://odl.readthedocs.io/dev/document.html